admin/config.yml

```
@@ -222,9 +222,23 @@ collections:
         hint: "Format : Prénom Nom. Ton nom complet est utilisé pour le nom de ta fiche. Attention à ton orthographe !"
         name: "fullname"
         widget: "string"
-      - label: "Rôle"
-        hint: "Développeuse ? Intrapreneur ? Chargée de développement ? Coach ?"
-        name: "role"
+      - name: domaine
+        hint: dans quel domaine principal intervenez-vous ?
```

pour l'homogénéité > "Dans quel domaine principal interviens-tu ? "

---

tu as une icone "suggestion" quand tu fais un commentaire 
Par exemple :
```suggestion
        hint: Dans quel domaine principal interviens-tu ?
```